load-testing-examples/test_test_creation.py

```diff
@@ -106,4 +144,69 @@ def test_test_creation(sdk: yandexcloud.SDK, create_agent_for_test):
     get_report_request = GetTableReportRequest(test_id=test_id)
     report: GetTableReportResponse = report_stub.GetTable(get_report_request)
 
-    assert 200 in report.overall.http_codes
\ No newline at end of file
+    assert 200 in report.overall.http_codes
+
+
+def test_test_interrupt(sdk: yandexcloud.SDK, create_agent_for_test):
+    agent_id = create_agent_for_test.id
+    config_stub: ConfigServiceStub = sdk.client(ConfigServiceStub)
+    create_config_operation = config_stub.Create(
+        CreateConfigRequest(
+            folder_id=ENV_VARS.folder_id(),
+            yaml_string=TEST_CONFIG_TEMPLATE.format(target_ip=ENV_VARS.target_IP()),
+        )
+    )
+    config_id = sdk.wait_operation_and_get_result(
+        create_config_operation,
+        response_type=Config,
+        meta_type=CreateConfigMetadata,
+        timeout=60,
+    ).response.id
+
+    create_test_request = CreateTestRequest(
+        folder_id=ENV_VARS.folder_id(),
+        configurations=[
+            SingleAgentConfiguration(
+                config_id=config_id, agent_selector=AgentSelector(agent_id=agent_id)
+            )
+        ],
+        test_details=Details(name='ete_created'),
+    )
+
+    test_stub: TestServiceStub = sdk.client(TestServiceStub)
+    create_test_operation = test_stub.Create(create_test_request)
+    test_id = sdk.wait_operation_and_get_result(
+        create_test_operation,
+        response_type=Test,
+        meta_type=CreateTestMetadata,
+        timeout=60,
+    ).response.id
+
+    get_test_request = GetTestRequest(test_id=test_id)
+    for seconds in range(3 * 60):
+        test: Test = test_stub.Get(get_test_request)
+        if test.summary.status != Status.CREATED:
+            break
+        time.sleep(1)
+    else:
+        raise Exception(f'can\'t wait for test finishing anymore. Waited {seconds=}')
+
+    test_stub.Stop(StopTestRequest(test_id=test_id))
```

Comment:
I think it's more idiomatic to wait for operation to finish and then single request to api to check status

Reply:
changed the code so that a dedicated sdk method is used for operation waiting